src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -1765,6 +1766,9 @@ private function export($value)
 
     private function doExport($value, $resolveEnv = false)
     {
+        if ($resolveEnv && \is_string($value) && array_key_exists($value, $this->exportedVariables)) {
```

Comment:
result of `$resolveEnv && \is_string($value)` can be saved into variable so it doesn't need to be evaluated later again